sample.env

```
@@ -12,11 +12,16 @@ CELERY_BROKER_URL=redis://127.0.0.1:6379/1
 # S3 Bucket for HMRC envelope uploads.
 HMRC_STORAGE_BUCKET_NAME=hmrc
 HMRC_STORAGE_DIRECTORY=tohmrc/staging/
-LOADING_REPORTS_BUCKET_NAME=loading_reports
 AWS_ACCESS_KEY_ID=minio_access_key
 AWS_SECRET_ACCESS_KEY=minio_secret_key
 AWS_S3_ENDPOINT_URL=http://localhost:9003
 
+# S3 packaging automation.
+HMRC_PACKAGING_SEED_ENVELOPE_ID=230000
```

If we don't need the year portion of this ID (as previous comment), then:
```suggestion
HMRC_PACKAGING_SEED_ENVELOPE_ID=0001
```

---

Ah I missed this we can update that and the default in the settings

---

Yeah, go for it.